src/client/src/components/App.js

```diff
@@ -110,12 +110,12 @@ class App extends Component {
                     />
                     <Route
                       exact
-                      path="/participants/course/details"
+                      path="/participant/:id/course/details/:id"
```

Comment:
I think you can't use `:id` twice for it to be parsed into `this.props.match.params` properly in the component, so maybe you could do `path="/participant/:id/course/details/:course_id"`

Reply:
I didn't have problems with it because I didn't use them :smile: 
soo, yeah I got it now :+1: